### PR TITLE
Test env checks

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: pytest_pootle/env.py [hunk 1]
```diff
@@ -428,7 +428,7 @@ def _add_submissions(self, unit, created):
 
         # add another suggestion as different user 7 days later
         suggestion2, _ = unit.add_suggestion(
-            "Suggestion 2 for %s" % unit.source,
+            "Suggestion 2 for %s" % (unit.target or unit.source),
```
Comment (unho) [inline pytest_pootle/env.py]: Is this change actually necessary?

Comment (phlax) [inline pytest_pootle/env.py]: yes - because otherwise the (erroneous) target string is overwritten with (not-erroneous) source

**Comment by unho** [comment]
lgtm
